app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id_id
+  has_many :comments, foreign_key: :user_id
+  has_many :likes, foreign_key: :user_id
```

Comment:
- [ ] Kindly correct the ID naming for the author on line two (2), it should be `author_id` I guess, and not `author_id_id`.
- [ ] There will be a conflict on the ID for the foreign key on line three (3) and line four (4) because in the likes and comments tables (models) the reference is `author` and not `user`. Kindly reference the `author_id` which is the known id for a user.